### DISCUSSION TITLE
推薦就讀中央大學嗎

### DISCUSSION THREAD

**Discussion by lai1008** [1101-GS4719-C/Python-Community-GS4719-C#2244] [General]
想問遺下各位會不會推薦自己的學弟妹來就讀中央?

**Comment by zoo100130** [comment]
看他分數到哪裡還有想走的領域吧
中央我覺得跟其他學校最不同的大概就是全台灣只有中央有太空系。
不然其實地理環境沒什麼優勢、交通也不用說了、生活機能一言難盡。

**Comment by gigitilto** [comment]
我推薦來中央就讀，中央的學生都很善良，腳踏車不會被偷，冰箱的食物每次都可以保存的很好，宿舍也很乾淨，洗手台不會被倒食物殘渣，廁所浴室十分舒服，至於每個科系的系館都不太一樣，但科四館是真的不錯，非常推薦入學。

**Comment by Boying12** [comment]
當然會推薦學弟妹來中央大學就讀！中央大學擁有優秀的教學資源和豐富的學術氛圍，讓學生能夠在這裡獲得全面的知識與技能。學校的師資陣容非常強大，許多教授在各自的領域內都具備相當的專業背景，能提供學生寶貴的學習經驗。
此外，中央大學的校園環境也十分優美，擁有豐富的自然景觀，非常適合學習和生活。在這裡，學生不僅能享受到良好的學習氛圍，還能參加各種社團活動和校園活動，增進人際關係，培養團隊合作精神。
更重要的是，中央大學的校友網絡非常強大，畢業生在各行各業都表現優異，這對於未來的職業發展有很大的幫助。總之，我非常推薦學弟妹來中央大學就讀，相信他們一定會在這裡有一段充實而愉快的學習生活！

**Comment by Chattenoire** [comment]
其他系不知道，但是工二館的系所千萬不要就讀；

同學絕對不會惡性內捲、教授絕對不會依老賣老/我行我素、系所絕對不是只有一條路可以走、絕對不是快沒有其他組的新老師、絕對不是出了一個15年毫無作為的校長、絕對沒有因為高階行政職都被校長換成系上的導致其他系資源被限縮、絕對沒有畢業制度死板導致難以跨領域。

不要不聽，真的不聽進來就知道。

**Comment by yuan3x2** [comment]
我認為只要是適合自己的學校，能夠找到適應方法，我都是推薦的，畢竟每間學校都有自己的優劣，至於中央大學的話，我認為不算太差，環境也不會特別糟糕，如果是想出國刷GPA的話非常推薦。

**Comment by lai1008** [comment]
我非常推薦就讀中央大學，中央的學生都很善良，教授人也都很好，冰箱的食物每次都可以保存的很好，宿舍也很乾淨而且便宜，洗手台不會被倒食物殘渣，廁所浴室十分舒服，至於每個科系的系館都不太一樣，但科五館是真的不錯，非常推薦入學。

**Comment by qiu0109** [comment]
我覺得分數有到就可以過來，中央的環境雖然說不上很好，但也不會太差，真的就是"中央"。反正就算去了其他學校還是會遇到其他缺點，不如選一個以自己能力來說可以讀到的最好的大學，以免選了以後後悔當初怎麼不去分數高一點的學校。

**Comment by 113607514** [comment]
如果問我會不會推薦學弟妹來就讀中央，我的答案是會的。雖然每所學校都有其特色和挑戰，但中央提供的資源和環境讓我覺得很值得，尤其是課程設計和師資力量，對我們的學習幫助很大。而且這裡的社群也蠻活躍的，雖然有時候感覺課業壓力不小，但能在這樣的氛圍中學到更多，不過還是得看每個人的需求和喜好。如果學弟妹想要一個有挑戰也有成長機會的環境，那中央絕對是個不錯的選擇！

**Comment by ed112601513** [comment]
如果能接受吃東西的選擇比較少，和生病都要跑個老遠，我是推薦可以來中央的，我覺得中央學生的素質人品都還不錯，就算是不小心考進來的也可以選擇台聯大的轉學，如果中央有學弟妹想要讀的科系，而且分數又剛好可以上的話，讀中央大學其實不差。

**Comment by Elaine4488** [comment]
學校的選擇還是看每個人的需求與目標，若是他的專業方向和興趣跟學校的資源匹配，我覺得我還是會推薦，雖然讀下來覺得缺點蠻多，但到現在也不覺得有後悔過，其實去每個地方都一定有好有壞，很多是我們沒去讀的人感受不到的。

**Comment by SendaiHatsuki** [comment]
完全不會想推薦別人來讀，學校小就算了，東西能吃的也很少，更不用說娛樂。而且我們系上的教授一直跳去頂大，課都開不齊，有夠慘完全跟頂大沒有可比性。

**Comment by XiangLuen** [comment]
我不會很推薦，雖然學校算是大，但實際上沒有其他特別突出，像是校內的學餐賣得很貴，沒有考慮學生感受，而到後門感覺就像到雙北，貴又不是很讓人喜歡。另外，校內的教授雖然都蠻好的，但是因為系辦的課程編排很爛，導致學生沒有辦法修讀想修的課程，像某系就推你可以選某專長幾學分就可以用這領域畢業，結果那專長課幾乎都不開或是很早就停了，導致學生無法透過這畢業，還要另外轉跑道，非常不開心

**Comment by lichilun** [comment]
如果有學弟妹考慮目標是在中央，我會推薦他們來讀，雖然中央食物少、地理位置超偏、課程無法開設齊全，但也有吸引人的地方，例如：松鼠、社團多元性、85折全家(xddd。而且我相信每一間大學都有優點和缺點，因此最重要的事，還是要了解自己的目標和方向，如果中央大學的教學資源符合你需求，那我覺得可以來中央就讀，給自己成長的機會。

**Comment by a414414** [comment]
愛中央、、如果你不喜歡中央，那請你趕緊離開。這裡的紀念品實在太多了，怕你錢包忙不過來，每天都得面對被偷的雨傘、腳踏車、學生證這些奢侈的煩惱；這裡的人才太多了，怕你NCU Talk板不夠用，從上課施工的道路到暑假不修開學才修的羽球館，每一處都美得讓人心醉；這裡的人太熱情了，怕你心臟受不了，道路周圍都有精美的測速器、、就怕你撿不到松果無法享受飆速的快感。
你離開後，可以去一個沒有這些"麻煩"的地方，過你的"平凡"生活，慢走、、不送、、

**Comment by lilipop6405** [comment]
中央大學非常吸引人! 天氣很好，風大到可以把你吹走，十分涼爽。交通便利，只有216、232可以到中壢市區，然後232又等很久，可以體驗等車樂趣。再來還有吃的，有兩條路可以吃，有選擇障礙的推薦來讀中央，不像中原有很多東西吃，會浪費很多時間在挑食物。

**Comment by mattwang1222** [comment]
不夠自律的人很適合，因為這裡不像台北交通方便娛樂發達，大部分時間學生都選擇待在學校附近活動。

**Comment by Kaz9487** [comment]
看人，這裡不適合對生活品質有要求的人來。如果對食物要求不高，也沒什麼娛樂需求，歡迎報考國立中央大學。但如果成績能上成大同科系，只是不想離家遠的話，還是推薦去成大

**Comment by hollowha** [comment]
國立中央大學是一所積極新創、學科齊全、學術實力雄厚、辦學特色鮮明，在桃園市具有重要影響力與競爭力的大學，在多個學術領域具有非常前瞻的科技實力，擁有世界一流的實驗室與師資力量，各種排名均位於全球前列。歡迎大家報考國立中央大學。

**Comment by yuuu3957** [comment]
如果就中央在業界或學界的名聲、排名而已，對於理工科學生算是一個好學校，資源感覺也很多，再加上台聯大系統，感覺就對學習知識、技能而言，中大是個好學校。
但我覺得就生活品質而言，其實沒有很好，個人已經算是很不看重食物跟生活品質的人，但還是覺得這邊不太方便，再加上氣候的部分，中大附近的溼氣真的好重，冬天又好冷，會讓人也不由自主的感覺喪喪的。
會不會推薦的部分，感覺還是得看學弟妹的個人偏好吧，畢竟本來每一所學校都有其優缺點。

**Comment by timlin940** [comment]
中央的資源還是蠻多的，推薦(死天氣除外)

**Comment by roy79231** [comment]
非常推薦，唯一缺點是沒啥好吃的

**Comment by Twicky123** [comment]
我會推薦就讀中央大學，因為我覺得中央的生活環境還蠻不錯的，也有很多很好的老師!

**Comment by yurukung** [comment]
雖然嫌棄中壢的天氣跟交通環境，但嫌歸嫌，其實蠻有歸屬感，也蠻喜歡這裡的。不會說推薦或不推薦，就是適合個人的就是最好的

**Comment by shrshu38** [comment]
我很推薦學弟妹來中央大學就讀！這裡不只有專業的師資和豐富的學習資源，校園環境也很美，適合讀書和生活。課業之外還有很多社團活動可以參加，能交朋友、培養軟實力。整體來說，是個充實又值得的學習環境。

**Comment by jaychang1027** [comment]
如果你本身喜歡清幽一點的地方的話其實中央還不錯，晚上很適合出去散散步，不過附近是真的沒有甚麼吃的，基本上是美食沙漠，加上冬天的時候這裡的天氣真的是非常的破，風很大又很冷

**Comment by 12388833** [comment]
推薦就讀中央大學，因為它好歹也是中字輩。中央大學算在山上，適合專心念書。學術方面，中央大學在理工、地科、資工、企管等領域具有良好口碑，且與國內外多所大學及研究機構有合作機會，對未來升學或就業都有幫助。整體而言，中央大學是一所兼具學術深度與生活品質的國立大學，值得考慮就讀。

**Comment by yunnnnchieh** [comment]
感覺如果是以學術角度的話，非常因人而異，可能還蠻看自己的造化的，但如果用自然角度看的話，如果不要有毛毛蟲雨跟把雨傘吹壞的怪風的話，天氣好的時候，感覺校園自然環境算是很不錯了，樹跟花跟星星都很漂亮

**Comment by ichenx2** [comment]
我覺得要看科系欸，像我是管院的，老實說如果有更好的選擇我可能會建議去別的學校，但如果分數剛好落在中央，其實也不差啦，校名還不錯聽，地點也靠近北部，對之後找實習會比中南部學校方便。認真說學校資源其實蠻多的，只是需要自己主動去找。缺點的話我覺得地理位置真的不太方便，吃的很普通，然後十天可能有九天在下雨。
